src/main/java/racingcar/Race.java

```diff
@@ -0,0 +1,39 @@
+package racingcar;
+
+public class Race {
+
+    private static final int MAX_NUMBER = 10;
+    private static final int MIN_NUMBER_FOR_START = 4;
+
+    public static int getMaxScore(Car[] cars) {
+        final int CARS_COUNT = cars.length;
+        int maximum = 0;
+        for (int i = 0; i < CARS_COUNT; i++) {
+            maximum = Math.max(maximum, cars[i].getPosition());
+        }
+        return maximum;
+    }
+
+    public static void repeatGame(Car[] cars, int times) {
+        for (int j = 0; j < times; j++) {
+            for (int i = 0; i < cars.length; i++) {
+                int randomNumber = getRandomNumber();
+                Race.compareWithFour(cars, i, randomNumber);
+            }
+            IOClass.printCurrentPosition(cars);
+            System.out.println();
+        }
+    }
+
+    static int getRandomNumber() {
+        int randomNumber = (int) Math.floor(Math.random() * MAX_NUMBER);
+        return randomNumber;
```

Comment:
불필요한 변수 선언인 것 같아요. 👀
```suggestion
        return (int) Math.floor(Math.random() * MAX_NUMBER);
```